.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+on: 
+  pull_request:
+    branches: master
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Okteto Context
+      uses: okteto/context@latest
+      with:
+        url: ${{ secrets.OKTETO_PRODUCT_URL }}
+        token: ${{ secrets.OKTETO_PRODUCT_TOKEN }}
+
+    - name: Login to DockerHub
```

Comment:
We could define these credentials in product.okteto.dev, but that would give us push permissions when running `okteto build` locally. I think it's more secured to define these credentials as GiHub secrets